plugins/linear-comparative-view/src/LGVSyntenyDisplay/model.ts

```diff
@@ -166,10 +166,51 @@ function stateModelFactory(schema: AnyConfigurationSchemaType) {
         },
       }
     })
+    .views(self => {
+      const superTrackMenuItems = self.trackMenuItems()
+      const toFind = [
+        'Show soft clipping',
+        'Sort by',
+        'Fade mismatches by quality',
+        'Color scheme',
+      ]
+
+      toFind.forEach((target: string) => {
```

Comment:
It might be good to avoid this type of filtering. this is sort of related to discussion during meeting yesterday that 'composing' can be better than filtering. to use composition to build up the list, you can perhaps use a "mixin" that adds coloring methods can be used, and the linerapileupdisplay and the lgvsyntenydisplay can choose separate sets of mixins

Reply:
for balance, doing this with mixins is a more extensive refactor